src/Controls/src/Build.Tasks/XamlCTask.cs

```diff
@@ -151,6 +151,8 @@ public class XamlCTask : XamlTask
 		/// </summary>
 		public bool ValidateOnly { get; set; }
 
+		internal bool GenerateFullILInValidateOnlyMode { get; set; }
```

Comment:
So, we shouldn't have a need to ever set this from MSBuild? Just unit tests?

Reply:
Yes, exactly.